README.md

```diff
@@ -17,7 +17,7 @@ I will be building my website portfolio during the first module of the program.
 - [x] Project 5: Application deploy
 - [x] Project 6: Evaluate accessibility (pair programming)
 - [x] Project 7: Mobile menu (pair programming)
-- [ ] Project 8: Details modal (pair programming)
+- [x] Project 8: Details modal (pair programming)
```

Comment:
:fire:

Reply:
I do not understand this. Could you, please, clarify.

Reply:
I'm sorry. I was just praising the way you're organizing your README with milestones and checks! :slightly_smiling_face: